frontend/assets/stylesheets/components/_event-listings.scss

```diff
@@ -0,0 +1,128 @@
+/* ==========================================================================
+   Event Listings
+   ========================================================================== */
+
+/* ==========================================================================
+   Event Listings - Header Bar
+   ========================================================================== */
+
+.header-bar {
+    position: relative;
+    padding-left: rem($gs-gutter / 2);
+    padding-right: rem($gs-gutter / 2);
+
+    .header-bar__title {
+        @include fs-headline(4);
+        padding: rem($gs-gutter / 2) 0 rem(55px);
+        max-width: rem(gs-span(8));
+    }
+
+    @include mq(tablet) {
+        min-height: rem(132px);
+        padding-left: rem($gs-gutter);
+
+        .header-bar__title {
+            @include fs-headline(6);
+            padding: rem($gs-gutter) 0;
+        }
+    }
+
+    @include mq(mem-full) {
+        padding-left: rem(gs-span(2) + $gs-gutter * 2); // 180
```

Comment:
This probably isn't your work but that comment is a bit pointless IMO, delete it?